src/feature.js

```diff
@@ -0,0 +1,2 @@
+/* eslint-disable import/prefer-default-export */
+export const checkIfDone = (done) => !done;
```

Comment:
Your module file is **incomplete**.  It should contain the following:

- [ ] Methods related to the status updates (completed: true / false).
- [ ] Event listener on the checkbox (change).

Reply:
The status update is explicitly reflected by how a task item can be marked as done or not done, as seen in how the strikethrough being is applied. The status of isDone literally functions the same way as isCompleted, meaning it updates the status just fine.